### ISSUE TITLE
logging

### ISSUE THREAD

**Issue by uatach** [numerai/numerapi#8]
I find it kind of weird to pass a verbosity level on NumerAPI constructor, usually for a lib is recommended to attach a NullHandler [1] and let the consumer configure it.

[1] https://docs.python.org/3.6/howto/logging.html#configuring-logging-for-a-library

**Comment by uuazed** [comment]
Yes, I agree that the current logging setup is suboptimal. Thanks for the pointer to that doc, looks like the way forward.
